ParkingLot/ParkingLot.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Text;
+
+namespace ParkingLot
+{
+    public class ParkingLot
+    {
+        public ParkingLot(string lotName)
+        {
+            LotName = lotName;
+        }
+
+        public string LotName { get; }
+        public int Capacity => 10;

```

Comment:
capacity may be changeable in the future

Reply:
Yes, I suggest to set the capacity in the constructor method